package.json

```
@@ -118,5 +119,8 @@
       "npm-funding": true
     }
   },
-  "types": "./index.d.ts"
+  "types": "./index.d.ts",
+  "dependencies": {
+    "yarn": "^1.22.22"
+  }
```

this does not look right

```suggestion
  "types": "./index.d.ts"
```